mdstcpip/mdsipshr/GetMdsMsg.c

```diff
@@ -121,13 +121,15 @@ Message *GetMdsMsgTOC(Connection *c, int *status, int to_msec)
     {
       Message *m;
       memcpy(&msglen, msg->bytes, 4);
-      dlen = msglen - sizeof(MsgHdr);
```

Comment:
What does moving this change ?

Reply:
`msglen` (as in **uncompressed** msglen) is potentially byte swapped, so it should not be used before byte swapping is handled.

Reply:
As Timo said, there is the chance that there could be a byte swapped before it is used.